src/index.js

```diff
@@ -0,0 +1,120 @@
+import './style.css';
+import { Todo, TodoLoad } from './Add_remove.js';
+import { IsChecked, clearAllCompleted } from './completed.js';
+
+let todos = [];
+
+const MyTodoList = document.getElementById('TodoList');
+
+MyTodoList.addEventListener('click', (e) => {
+  const task = new Todo();
+  if (e.target.nodeName === 'INPUT') {
+    e.target.parentElement.parentElement.style.backgroundColor = '#f6e4672e';
+    e.target.parentElement.parentElement.children[1].style.display = 'none';
+    e.target.parentElement.parentElement.children[2].style.display = 'inline';
+    setTimeout(() => {
+      e.target.parentElement.parentElement.style.backgroundColor = '#fff';
+      e.target.parentElement.parentElement.children[1].style.display = 'inline';
+      e.target.parentElement.parentElement.children[2].style.display = 'none';
+    }, 2000);
+  }
+  if (e.target.classList.contains('hidden')) {
+    task.deleteTodo(e, todos);
+  }
+});
+const showTodo = (todos) => {
+  MyTodoList.innerHTML = '';
+  for (let i = 0; i < todos.length; i += 1) {
+    if (todos[i].isComplete) {
+      MyTodoList.innerHTML += `<li class="todo_item" data-id="${todos[i].index}" id=${i} >
+    <div class="description">
+      <input type="checkbox" name="todo" class="toggle-check" checked />
+      <input value="${todos[i].description}" disabled/>
+    </div>
+    <i class="fa-solid fa-ellipsis-vertical "></i>
+    <i class="fa-solid fa-trash-can hidden"></i>
+  </li>`;
+    } else {
+      MyTodoList.innerHTML += `<li class="todo_item" data-id="${todos[i].index}" id=${i} >
+        <div class="description">
+          <input type="checkbox" name="todo" class="toggle-check" />
+          <input value="${todos[i].description}" disabled/>
+        </div>
+        <i class="fa-solid fa-ellipsis-vertical"></i>
+        <i class="fa-solid fa-trash-can hidden"></i>
+      </li>`;
+    }
+  }
+};
+
+const initAll = (input) => {
+  input.value = '';
+};
+
+const TodoCreate = (input) => {
+  const desc = input.value;
+  const task = new Todo(desc);
+  task.addTodo(task, todos);
+};
+
+const description = document.querySelector('#new-task');
```

Comment:
It is better to put the variable declaration at the top of your script.
please kindly move `const description = document.querySelector('#new-task');`  to the top. Because if you put it at the top it is easy to find out the variable if you need to update latter.